.github/workflows/dependabot.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       run: make modules
     - name: Update generated code
       run: make generate
-    - uses: EndBug/add-and-commit@v7
+    - uses: EndBug/add-and-commit@8c12ff729a98cfbcd3fe38b49f55eceb98a5ec02 # v7
```

Comment:
How can we look up those sha's?

Reply:
I have included the URL to the commit to check if they are correct. HTH